.changeset/techdocs-fifty-cameras-listen.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs': patch
+---
+
+Only update the `path` when the content is updated.
```

Comment:
Thanks @dhenneke!  Would you mind updating with a note describing the flickering issue resolved by this?  Will be helpful when looking back through release notes in the future.